multi-agent/lib/mas/elements/providers/mcp_server_client/transport/sse_transport.py

```
@@ -7,12 +7,28 @@
+import httpx
 from mcp.client.sse import sse_client
 
 from .base_transport import BaseTransportManager
 from .enums import McpTransportType
 
 
+def _create_httpx_client(
+    headers=None, timeout=None, auth=None,
+) -> httpx.AsyncClient:
+    """httpx client factory that skips SSL verification and forces IPv4."""
+    transport = httpx.AsyncHTTPTransport(verify=False, local_address="0.0.0.0")
```

Ruff (0.15.20)

[error] Possible binding to all interfaces

(S104)